frontend/src/pages/Admin/Components/Setting/Manage/Manage.jsx

```diff
@@ -1,16 +1,27 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import style from "./Manage.module.scss";
 import DeleteIcon from "@material-ui/icons/Delete";
+import { useSelector } from "react-redux";
 
 export function Manage() {
-  const [data, setData] = useState([
-    "Abhishek",
-    "Aman",
-    "Dikshit",
-    "Adarsh",
-    "Utkarsh",
-    "Vatan",
-  ]);
+  const token = useSelector((state) => state.token);
+
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    fetch("http://localhost:3500/admin", {
```

Comment:
instead of hard-coding the endpoint everywhere, can we please have it in a single folder. Maybe name it as `config`

Reply:
ok got it! will do it and update the PR